app/navigation/JourneyNavigator.scala

```diff
@@ -44,17 +46,23 @@ class JourneyNavigator @Inject()() extends Navigator {
 
     // Income from property, pensions and investments
     case RentalIncomePage                    => ua => controllers.propertypensionsinvestments.routes.PensionsController.onPageLoad(NormalMode, ua.taxYear)
-    case PensionsPage                        => ua => controllers.propertypensionsinvestments.routes.UkInsuranceGainsController.onPageLoad(NormalMode, ua.taxYear)
+    case PensionsPage                        => ua => controllers.propertypensionsinvestments.routes.UkInsuranceGainsController.
+      onPageLoad(NormalMode, ua.taxYear)
     case UkInsuranceGainsPage                => ua => controllers.propertypensionsinvestments.routes.UkInterestController.onPageLoad(NormalMode, ua.taxYear)
-    case UkInterestPage                      => ua => controllers.propertypensionsinvestments.routes.UkDividendsSharesLoansController.onPageLoad(NormalMode, ua.taxYear)
+    case UkInterestPage                      => ua => controllers.propertypensionsinvestments.routes.UkDividendsSharesLoansController.
+      onPageLoad(NormalMode, ua.taxYear)
     case UkDividendsSharesLoansPage          => ua => routes.AddSectionsController.onPageLoad(ua.taxYear)
 
-
     // Payments into pensions
     case PaymentsIntoPensionsPage             => ua => routes.AddSectionsController.onPageLoad(ua.taxYear)
 
+    //Income from work and taxable state benefits section
+    case AboutYourWorkRadioPage               => ua => controllers.workandbenefits.routes.ConstructionIndustrySchemeController.
+      onPageLoad(NormalMode, ua.taxYear)
```

Comment:
If  you need to split the line up because it is too long I would suggest splitting it after the second =>.

Similar to how they are in the "ukResidenceStatusRoute" function.